test/Amazon.AspNetCore.DataProtection.SSM.Tests/Amazon.AspNetCore.DataProtection.SSM.Tests.csproj

```diff
@@ -1,27 +1,30 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-
-    <IsPackable>false</IsPackable>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <Compile Include="..\..\src\Amazon.AspNetCore.DataProtection.SSM\ExtensionMethods.cs" Link="LinkedFiles\ExtensionMethods.cs" />
-    <Compile Include="..\..\src\Amazon.AspNetCore.DataProtection.SSM\PersistOptions.cs" Link="LinkedFiles\PersistOptions.cs" />
-    <Compile Include="..\..\src\Amazon.AspNetCore.DataProtection.SSM\SSMXmlRepository.cs" Link="LinkedFiles\SSMXmlRepository.cs" />
-    <Compile Include="..\..\src\Amazon.AspNetCore.DataProtection.SSM\SSMParameterToLongException.cs" Link="LinkedFiles\SSMParameterToLongException.cs" />
-    <Compile Include="..\..\src\Amazon.AspNetCore.DataProtection.SSM\TierStorageMode.cs" Link="LinkedFiles\TierStorageMode.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
-	<PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.0.1" />
-	<PackageReference Include="AWSSDK.SimpleSystemsManagement" Version="3.7.0.2" />
-    <PackageReference Include="Microsoft.AspNetCore.DataProtection.Extensions" Version="2.0.3" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />
-    <PackageReference Include="Moq" Version="4.8.2" />
-    <PackageReference Include="xunit" Version="2.3.1" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.1" />
-  </ItemGroup>
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>

```

Comment:
To be clear I think the library needs to target .NET 3.1 but the tests can be on .NET 6. Thanks for getting that updated.